src/components/charts/bar-chart.js

```diff
@@ -13,6 +13,7 @@ import chartStyles from './charts.module.scss'
 import styles from './bar-chart.module.scss'
 
 import colors from '~scss/colors.module.scss'
+import { renderedComponent } from '~plugins/gatsby-render-components'
```

Comment:
Let's remove the render stuff from the bar charts - we're going to probably want to build entirely separate parent components that include bar charts within them.

Reply:
Done! Makes sense. Do you have an idea of how you want this to be done or how you want the end result to look? Just cuz there may be some extra work needed around props passing while rendering depending on implementation.